### PR TITLE
Created a decision record for not using a forked repo's GitHub pages …

### LINKED ISSUE #6219: Instructions for Viewing the Website through a Forked Repo's GitHub Pages
### Overview
As developers, we need instructions for how to view code changes to the website through a forked repo using GitHub Pages so that developers who are unable to use Docker to view code changes can have another option.

#### Developer
- [x] In a new comment, write instructions for how to view code changes through a forked repo using GitHub pages and add a link to in [^1] in the _Resources/Instructions_ section
  - [x] Include possible pros and cons of this method

#### Tech Lead
- [x] Read through the instructions and decide 
  - [x] 1) If we want to allow developers to use this method
  - [x] 2) Depending on the decision for 1, determine the next steps for this issue, such as whether to add to website-wiki or CONTRIBUTING.md, closing the issue as not planned and add a decision record to the website-wiki, etc

### Resources/Instructions
- https://pages.github.com/
- [Graphics and Annotations for Issue 6219](https://docs.google.com/presentation/d/1i8D4iLchqgWcYZczD5VwKXcelz3UDFAEFj1xOj0jLvc/edit#slide=id.gcd84038bdd_0_0) starting with slide 12
[^1]: Link to comment with instructions: https://github.com/hackforla/website/issues/6219#issuecomment-1925670289

### PR DESCRIPTION
**Pull request by JessicaLucindaCheng**
Fixes https://github.com/hackforla/website/issues/6219

### What changes did you make?
  - I wrote and added the decision record for not using a forked repo's GitHub Pages to view the website. The file added was using-a-forked-repos-github-pages-to-view-the-website.md in the docs/decision-records/not-implemented directory.

### Why did you make the changes (we will use this info to test)?
  - I wrote the decision record so we have a record of the decision made to not use GitHub Pages to view the website and the reasons behind this decision.

### Screenshots of Proposed Changes To The Website (if any, please do not include screenshots of code changes)
 

<details>
<summary>Visuals before changes are applied</summary>
Not applicable because the file did not exist before.
</details>

<details>
<summary>Visuals after changes are applied</summary>

DR-not-implemented-using-a-forked-repos-github-pages-to-view-the-website

</details>

### CODE REVIEW

File: docs/decision-records/not-implemented/using-a-forked-repos-github-pages-to-view-the-website.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+#### Feasibility Determination
+
+We have made a decision not to do this for the following reasons:
+
+1. We are concerned that old versions of our site will remain out on people's forked repos and could eventually show up in search results, which would lower the quality of results people get when searching for the Hack for LA (HfLA) website.
+
+2. It's quite complicated to follow the instructions and requires the Merge Team to support people who don't understand the instructions, thus increasing the burden on the Merge Team.
+
+3. We had thought we needed it for work on the Wins page's Google Apps Script to test and see the visual changes on the website. However, for testing, a team member can initiate a commit to their own forked repo of the website in Google Apps Script, and then pull down the site from their own forked repo and see it using Docker.
```
Comment (JessicaLucindaCheng) [inline docs/decision-records/not-implemented/using-a-forked-repos-github-pages-to-view-the-website.md]: This was originally from https://github.com/hackforla/website/issues/6219#issuecomment-1972160104

> 3. We had thought we needed it for the wins page work, but team member who have to do a pr to their own repo of the website to test, can then just pull down the site from their own repo and see it in docker.

However, I don't think someone can create a PR directly in their own forked repo. Is the above more accurate as to what we want happening when testing the Wins page?

**Comment by anthonysim** [comment]
Hi, Sorry, I keep getting subscribed back when I remove myself from this subscription. Just wondering if I could get removed. Sincerely,Anthony SimSent from my iPhoneOn Sep 16, 2024, at 4:41 PM, Jessica Cheng ***@***.***> wrote:﻿
@JessicaLucindaCheng commented on this pull request.



In docs/decision-records/not-implemented/using-a-forked-repos-github-pages-to-view-the-website.md:
> +
+Some developers are unable to use Docker to view code changes.
+
+#### Potential Solution
+
+Provide instructions for how to view code changes to the website through a forked repository (repo) using GitHub Pages so that developers unable to use Docker can have another option.
+
+#### Feasibility Determination
+
+We have made a decision not to do this for the following reasons:
+
+1. We are concerned that old versions of our site will remain out on people's forked repos and could eventually show up in search results, which would lower the quality of results people get when searching for the Hack for LA (HfLA) website.
+
+2. It's quite complicated to follow the instructions and requires the Merge Team to support people who don't understand the instructions, thus increasing the burden on the Merge Team.
+
+3. We had thought we needed it for work on the Wins page's Google Apps Script to test and see the visual changes on the website. However, for testing, a team member can initiate a commit to their own forked repo of the website in Google Apps Script, and then pull down the site from their own forked repo and see it using Docker.

This was originally from hackforla/website#6219 (comment)


We had thought we needed it for the wins page work, but team member who have to do a pr to their own repo of the website to test, can then just pull down the site from their own repo and see it in docker.


However, I don't think someone can create a PR directly in their own forked repo. Is the above more accurate as to what we want happening when testing the Wins page?

**Comment by JessicaLucindaCheng** [comment]
@anthonysim 
> Hi, Sorry, I keep getting subscribed back when I remove myself from this subscription. Just wondering if I could get removed. Sincerely,Anthony SimSent from my iPhoneOn Sep 16, 2024, at 4:41 PM

Sorry, I'm unable to unsubscribe you from this repo. While you are logged into GitHub, you can go to https://github.com/watching, find this repo, and select "Ignore   Never be notified." option for it.

File: docs/decision-records/not-implemented/using-a-forked-repos-github-pages-to-view-the-website.md (hunk 1 above)
Comment (JessicaLucindaCheng) [reply docs/decision-records/not-implemented/using-a-forked-repos-github-pages-to-view-the-website.md]: Yes, at the Dev/PM meeting on 2024-09-23, I discussed this with Roslyn that the wording for point 3 as written in this PR is fine. However, I will change the wording "in Google Apps Script" to "in their copy of the Google Apps Script".

**Pull request merged**